File: src/main/java/me/miki/shindo/injection/interfaces/IMixinRenderPlayer.java
package me.miki.shindo.injection.interfaces;


import me.miki.shindo.management.mods.impl.skin3d.layers.BodyLayerFeatureRenderer;
import me.miki.shindo.management.mods.impl.skin3d.layers.HeadLayerFeatureRenderer;

public interface IMixinRenderPlayer {
	public boolean hasThinArms();
	public HeadLayerFeatureRenderer getHeadLayer();
	public BodyLayerFeatureRenderer getBodyLayer();
}

File: src/main/kotlin/me/miki/shindo/injection/mixin/interfaces/client/renderer/entity/IMixinRenderPlayer.kt
package me.miki.shindo.injection.mixin.interfaces.client.renderer.entity

import me.miki.shindo.management.mods.impl.skin3d.layers.BodyLayerFeatureRenderer
import me.miki.shindo.management.mods.impl.skin3d.layers.HeadLayerFeatureRenderer

interface IMixinRenderPlayer {
    fun hasThinArms(): Boolean

    fun getHeadLayer(): HeadLayerFeatureRenderer

    fun getBodyLayer(): BodyLayerFeatureRenderer
}
